scripts/detectUnpinnedVersions.fsx

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env -S dotnet fsi
+
+open System
+open System.IO
+
+#load "../src/FileConventions/Library.fs"
+
+let NotInDir (dirName: string) (fileInfo: FileInfo) =
+    not(
+        fileInfo.FullName.Contains
+            $"%c{Path.DirectorySeparatorChar}%s{dirName}%c{Path.DirectorySeparatorChar}"
+    )
+
+let invalidFiles =
+    Directory.GetFiles(".", "*.fsx", SearchOption.AllDirectories)
+    |> Seq.map(fun pathStr -> FileInfo pathStr)
+    |> Seq.filter(NotInDir "node_modules")
+    |> Seq.filter(NotInDir ".git")
+    |> Seq.filter(NotInDir "bin")
+    |> Seq.filter(NotInDir "obj")
+    |> Seq.filter(NotInDir "DummyFiles")
+    |> Seq.filter(fun fileInfo ->
+        not(FileConventions.DetectUnpinnedVersionsInGitHubCI fileInfo)
+    )
+
+if Seq.length invalidFiles > 0 then
+    let message =
+        "The following files shouldn't contain -latest in runs-on: GitHubCI tags:"
```

Comment:
@realmarv this is an err msg that is not clear enough, it doesn't give any hint on how to fix it; you should be crystal clear, even adding the URL to the list of image versions: https://github.com/actions/runner-images

Reply:
@realmarv note, I edited/updated my comment above

Reply:
Done